### PR TITLE
[BM-693] Update SwiftLint & fix new warnings

### PR DESCRIPTION
**Pull request by anyashka**
### Ticket

https://netguru.atlassian.net/browse/BM-693
 
### Task Description

 Updated SwiftLint and fixed most of the warnings except one, which been extracted to another task.

### CODE REVIEW

File: Baby Monitor/Source Files/Localizations/Localizable.swift [hunk 1]
```diff
@@ -217,6 +217,8 @@ enum Localizable {
     }
 }
 
+// swiftlint:disable nslocalizedstring_key
```
Comment (3Qax) [inline Baby Monitor/Source Files/Localizations/Localizable.swift]:
```suggestion
// swiftlint:disable:next nslocalizedstring_key
```

File: Baby Monitor/Source Files/Modules/Onboarding/Pairing/Multiple Devices/Code/OnboardingCompareCodeViewModel.swift [hunk 2]
```diff
@@ -62,7 +62,7 @@ final class OnboardingCompareCodeViewModel: BaseViewModel {
 
     private func saveEmptyStateIfNeeded() {
         let allActivityLogEvents = activityLogEventsRepository.fetchAllActivityLogEvents()
-        guard allActivityLogEvents.first(where: { $0.mode == .emptyState }) == nil else {
+        guard !allActivityLogEvents.contains(where: { $0.mode == .emptyState }) else {
```
Comment (3Qax) [inline Baby Monitor/Source Files/Modules/Onboarding/Pairing/Multiple Devices/Code/OnboardingCompareCodeViewModel.swift]:
```suggestion
        guard allActivityLogEvents.allSatisfy({ $0.mode != .emptyState }) else {
```

File: .swiftlint.yml [hunk 3]
```diff
@@ -48,7 +48,7 @@ opt_in_rules:
   - empty_string
   - identical_operands
   - inert_defer
-  - indentation_width
+#  - indentation_width #TODO in BM-694
```
Comment (3Qax) [inline .swiftlint.yml]: I don't have a strong preference here, but having two comment markers in one line seems weird.
```suggestion
# TODO in BM-694:   indentation_width
```

File: Baby Monitor/Source Files/Localizations/Localizable.swift (hunk 1 above)
Comment (3Qax) [reply Baby Monitor/Source Files/Localizations/Localizable.swift]: And you can delete line reenabling it below.

File: .swiftlint.yml (hunk 3 above)
Comment (anyashka) [reply .swiftlint.yml]: Thanks but I'll leave it as it is :)

**Pull request merged**